portingdb/templates/_base.html

```diff
@@ -195,6 +195,20 @@
     {% endif %}
 {%- endmacro %}
 
+{%- macro maintainer_link(maintainer) -%}
+    {% if maintainer.packages | length > 1 %}
+        <a href="{{ url_for('maintainer', name=maintainer.name) }}">
+            {{ maintainer.name }}
+        </a>
```

Comment:
this makes the space part of the link, maybe we can make this one line instead:

     <a href="{{ url_for('maintainer', name=maintainer.name) }}">{{ maintainer.name }}</a>

Reply:
Jinja has dashes for that :)